src/api/Versions.js

```diff
@@ -58,7 +64,7 @@ class Versions extends OffsetBasedAPI {
             return;
         }
 
-        this.successCallback(data);
+        this.successCallback(data || { id: this.versionId });
```

Comment:
It seems a little odd to always retain the original parameter and pass it back if the data payload is undefined. Can we use the original version id in the consuming call, instead?